<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>75</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>195</width>
    <height>75</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>195</width>
    <height>75</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Forbid sleeping mode</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>output_wnd.ico</normaloff>
    <normalon>output_wnd.ico</normalon>output_wnd.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnStartStop">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWorking">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="labelInterval">
     <property name="text">
      <string>Time interval: </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxInterval">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelTimeUnits">
     <property name="text">
      <string>minutes</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spinBoxInterval</tabstop>
  <tabstop>btnStartStop</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnStartStop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnStartStop_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnStartStop_clicked()</slot>
 </slots>
</ui>
